### DISCUSSION TITLE
Reporting for web address not working

### DISCUSSION THREAD

**Discussion by JyothiNagavelli** [Sunbird-Ed/Community#60] [Bugs] [answered]
Hi Team, 

I am trying to run the application to contribute to the sunbird product. however the instruction part of the web url is not working, so could you please look at this and send me some guidance that will help me to run the application on my laptop, thanks .

 here is the below url that not working 
https://www.sunbird.org/developer-docs/installation/

**Comment by sharathkashyap** [comment, marked as answer]
@JyothiNagavelli Kindly refer to the below link. As all the installation details are part of micro-site now. Request you to refer to the below link.
https://ed.sunbird.org/use/prerequisites-for-your-own-sunbird-ed-instance
